src/context/WalletProvider/KeepKeyProvider.tsx

```diff
@@ -90,14 +91,14 @@ const initialState: InitialState = {
 export interface IKeepKeyContext {
   state: InitialState
   setHasPassphrase: (enabled: boolean) => void
-  keepKeyWallet: KeepKeyHDWallet | undefined
```

Comment:
The crux of this PR. Remove the context `keepKeyWallet` prop and instead use the state set by the reducer.